tests/test_live_trader.py

```
@@ -204,3 +277,167 @@ def test_seed_demo_data():
+
+    engine._update_market_strategy(slug, poll_stop, now)
+    assert mstate.exit_taken is True
+    assert mstate.status == "STOP_EXIT"
+    # Verify open down order was cancelled
+    fake_client.cancel.assert_called_with("ord_active_dn")
+
+
+def test_live_order_flow_smoke(monkeypatch):
+    """Verify test_live_order_flow CLI runs successfully in dry-run mode."""
+    from scripts.test_live_order_flow import main
+    monkeypatch.setattr("scripts.test_live_order_flow.fetch_polymarket_account_value", lambda *a, **kw: {"success": True, "cash_balance": 100.0, "net_value": 100.0, "open_positions": 0})
```

Ruff (0.16.3)

[warning] Unused lambda argument: `a`

(ARG005)